extension/js/content_scripts/webmail/gmail-element-replacer.ts

```diff
@@ -438,6 +438,8 @@ export class GmailElementReplacer implements WebmailElementReplacer {
         }
       }
     }
+    console.log(attachmentsContainerInner);
+    console.log(attachmentsContainerInner.parent().find(this.sel.numberOfAttachmentsDigit));
```

Comment:
can be removed

Reply:
oops. forgot to remove. thanks for your notice